scripts/pi-hole/js/utils.js

```diff
@@ -13,11 +13,10 @@ function escapeHtml(text) {
     "&": "&amp;",
     "<": "&lt;",
     ">": "&gt;",
-    '"': "&quot;",
```

Comment:
Not sure about this removal. The `escapeHtml` function is used in plenty other places.

Reply:
I have another thought actually, we can leave the `"` in this function and adjust the php function in `groups.php` to replace `&quot;` with `"`, that should have the same effect. It's just that the groups function requires it.